backend/Dockerfile

```diff
@@ -10,5 +10,6 @@ COPY requirements.txt /backend/
 WORKDIR /backend
 RUN pip install -r requirements.txt
 EXPOSE 8000
+EXPOSE 8089
```

Comment:
`locust` *can* be run headless, but I think for our purposes the web-ui is kinda nice (so it needs a port) - just tested with locust port set to 8000, and got a error since that port is already in use, so imo probably fine to stick with default 8089